### PR TITLE
docs(journey-maps): add documentation for 'isLevelSwitchHorizontal'

### PR DESCRIPTION
**Pull request by swiss-chris**

### CODE REVIEW

File: src/journey-maps/angular/journey-maps.ts [hunk 1]
```diff
@@ -169,8 +169,11 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
   /** @docs-private */
   touchOverlayStyleClass: string = '';
 
+  /** In landscape mode, when the map height is below {@link SbbJourneyMaps#levelSwitchHorizontalThreshold},
```
Comment (mhaertwig) [inline src/journey-maps/angular/journey-maps.ts:172]: Nitpick, but please change the formatting to

```
/**
 * Text here
 */
```

Comment (swiss-chris) [reply src/journey-maps/angular/journey-maps.ts]: Done ✅

**Review by mstankala — APPROVED** [review]
LGTM

**Comment by swiss-chris** [comment]
This PR is being replaced by https://github.com/sbb-design-systems/sbb-angular/pull/2457

**Pull request closed without merging**